setupTests.js

```diff
@@ -20,3 +20,15 @@ jest.mock('react-intl', () => {
         useIntl: () => intl,
     };
 });
+
+jest.mock('mapbox-gl/dist/mapbox-gl', () => ({
+    GeolocateControl: jest.fn(),
+    Map: jest.fn(() => ({
+        addControl: jest.fn(),
+        on: jest.fn(),
+        remove: jest.fn(),
+        once: jest.fn(),
+    })),
+    NavigationControl: jest.fn(),
+}));
+jest.mock('mapbox-gl/dist/mapbox-gl.css', () => ({}));
```

Comment:
I think this can go in the map test file since this will not be used across all components and modules like `react-intl`, thus you can take more control on what you want to mock